beginner_source/audio_preprocessing_tutorial.py

```diff
@@ -0,0 +1,237 @@
+"""
+torchaudio Tutorial
+===================
+
+PyTorch is an open source deep learning platform that provides a
+seamless path from research prototyping to production deployment with
+GPU support.
+
+Significant effort in solving machine learning problems goes into data
+preparation. torchaudio leverages PyTorch’s GPU support, and provides
+many tools to make data loading easy and more readable. In this
+tutorial, we will see how to load and preprocess data from a simple
+dataset.
+
+For this tutorial, please make sure the ``matplotlib`` package is
+installed for easier visualization.
+
+"""
+
+import torch
+import torchaudio
+import matplotlib.pyplot as plt
+
+
+######################################################################
+# Opening a dataset
+# -----------------
+# 
+
+
+######################################################################
+# torchaudio supports loading sound files in the wav and mp3 format. We
+# call waveform the resulting raw audio signal.
+# 
+
+filename = "../_static/img/steam-train-whistle-daniel_simon-converted-from-mp3.wav"
+waveform, sample_rate = torchaudio.load(filename)
+
+print("Shape of waveform: {}".format(waveform.size()))
+print("Sample rate of waveform: {}".format(sample_rate))
+
+plt.figure()
+plt.plot(waveform.transpose(0,1).numpy())
```

Comment:
.t() seems cleaner than transpose(0,1)

Reply:
Indeed. I updated it in other places too.